### ISSUE TITLE
ER from TLDL: Combine Solidarity credits

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#4531]
TLDL stands for Task List Dev Leads.

### Emergent Requirement - Problem
**Issue Description**
Combine the Solidarity credit on the Credits page so that the same image from the same artist and provider but used on different web pages has only one credit

- Notes: 
  - If you look at the `image-url` in the credit files, one is a png format and one is a svg format. Is it possible to just use one of those image formats (such as the svg) on both the Donate and Getting Started page and use SCSS to style it to keep it looking the same on both pages as it is on the live website now? 
  - This could be a separate issue and would need to be completed before combining the Solidarity credits.

### Who was involved
- @JessicaLucindaCheng
- `@SAUMILDHANKAR`   

### What happens if this is not addressed
There are two Solidarity credits with the same image

### Resources
- Credits page: https://www.hackforla.org/credits/
- https://github.com/hackforla/website/blob/gh-pages/_data/internal/credits/solidarity.yml
- https://github.com/hackforla/website/blob/gh-pages/_data/internal/credits/donate.yml

### Recommended Action Items
- [x] Make a new issue
- [ ] Discuss with team
- [ ] Let a Team Lead know

**Comment by jdingeman** [comment]
Hey @JessicaLucindaCheng - so the .svg is credited as being using on the Donate page (and has a `-pana` in the filename, making me think it once was a panoramic on that page?)

Would it make sense to just get rid of the credit in /dontate.yml if it's not actually being used in it?

**Comment by jdingeman** [comment]
The [figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=24579-285100&t=87pXnsghMfgZcd1h-0) for the most recent update to the Donate page doesn't have the image in it anymore.

**Comment by JessicaLucindaCheng** [comment]
@jdingeman 

> Hey @JessicaLucindaCheng - so the .svg is credited as being using on the Donate page (and has a `-pana` in the filename, making me think it once was a panoramic on that page?)
> 
> Would it make sense to just get rid of the credit in /dontate.yml if it's not actually being used in it?

> The [figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=24579-285100&t=87pXnsghMfgZcd1h-0) for the most recent update to the Donate page doesn't have the image in it anymore.

Yes, I think deleting the donate.yml credit would make sense. 
Also, can the image (`/assets/images/donate/solidarity-pana.svg`) itself be deleted as well if it is not being used? Maybe this can ER can be 2 good first issues: 1) delete the donate.yml credit and 2) delete the svg?

**Comment by jdingeman** [comment]
Issues made here:
- https://github.com/hackforla/website/issues/4578
- https://github.com/hackforla/website/issues/4579

**Comment by ExperimentsInHonesty** [comment]
I don't want to suggest a course of action at this point.  But I still see https://github.com/hackforla/website/blob/gh-pages/_data/internal/credits/solidarity.yml and if the image is not being used anywhere, we need to remove it.  There might be other cleanup stuff to do...

**Comment by JessicaLucindaCheng** [comment]
> I don't want to suggest a course of action at this point. But I still see https://github.com/hackforla/website/blob/gh-pages/_data/internal/credits/solidarity.yml and if the image is not being used anywhere, we need to remove it. There might be other cleanup stuff to do...

- The _Solidarity_ image is used on the [Getting Started](https://www.hackforla.org/getting-started) page; it is in Step 2 in the header section of the page. So, there still should be one credit called _Solidarity_ on the [Credits](https://www.hackforla.org/credits/) page.
- I wrote this ER because there were two credits called _Solidarity_ with the same image at the time. Now, I only see one _Solidarity_ credit on the [Credits](https://www.hackforla.org/credits/) page, so I think the problem has been fixed.
